src/sentry/analytics/events/first_event_sent.py

```diff
@@ -19,9 +19,11 @@ class FirstEventSentEvent(analytics.Event):
 class FirstEventSentEventForProject(FirstEventSentEvent):
     type = "first_event_for_project.sent"
 
+    attributes = FirstEventSentEvent.attributes + (analytics.Attribute("project_platform"),)
+
 
 # first error with minified stack trace for a project
-class FirstEventSentEventWithMinifiedStackTraceForProject(FirstEventSentEvent):
+class FirstEventSentEventWithMinifiedStackTraceForProject(FirstEventSentEventForProject):
```

Comment:
Lmk if this doesn't make sense, but I figured it could use the new attribute too.

Reply:
@ceorourke makes sense to me! Just need to make sure this doesn't blow up because of an invalid parameter. Pretty sure we have a test though that would fail if we didn't do this right.